source/trackers.json

```diff
@@ -146,18 +188,60 @@
             "url": "https://canonical.com/",
             "companyId": "canonical"
         },
+        "channel_intelligence": {
+            "name": "Channel Intelligence",
+            "categoryId": 4,
+            "url": "http://www.channelintelligence.com/",
```

Comment:
It seems the most info is obsolete. Please use actual domains and add links with HTTPS protocol.